swift/ql/test/query-tests/Security/CWE-312/testNSUbiquitousKeyValueStore.swift

```diff
@@ -0,0 +1,70 @@
+
+// --- stubs ---
+
+class NSObject
+{
+}
+
+class NSUbiquitousKeyValueStore : NSObject
+{
+    class var `default`: NSUbiquitousKeyValueStore {
+        return NSUbiquitousKeyValueStore()
+    }
+
+	func set(_ value: Any?, forKey key: String) {}
```

Comment:
Yeah, except I assumed the naming of parameters doesn't really matter in stubs. The selector is the same, `set(_:forKey:)`, in both cases.

Reply:
As long as the query isn't looking at the name of the parameter, then yes, it shouldn't matter. But in case we ever decide to do that (for whatever reason that I cannot currently think of), I think we should stick to the same parameter names as used in real codebases.